jmin-jlink/agent/plugin.agent/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
+Created-By: 11.0.8-internal (N/A)
+Premain-Class: jmin.jlink.agent.MinimizeClassesPluginAgent

```

Comment:
This file should not contain carriage-return characters.

Reply:
It's just a text file and does not need special line-ends: they still appear to be `CR-LF` instead of just `LF`.

Reply:
Ah I see. I just removed the newline, but really I needed to change the file to use LF only. So I added the newline back in (I believe this is still needed? Maybe I'm wrong) and changed the file to use LF only.

Reply:
I'm actually not sure how it ended up being CRLF in the first place...maybe when I generated the JAR, the MANIFEST.MF created uses CRLF by default?